src/components/Calls/CallIFrame.js

```diff
@@ -7,8 +7,28 @@ import styles from './styles.scss';
 
 
 export const CallIframe = () => {
-	const { token, room } = store.state;
-	const url = `${ Livechat.client.host }/meet/${ room._id }?token=${ token }`;
+	const { token, room, incomingCallAlert } = store.state;
+	const url = `${ Livechat.client.host }meet/${ room._id }?token=${ token }&layout=embedded`;
```

Comment:
```suggestion
	const url = `${ Livechat.client.host }/meet/${ room._id }?token=${ token }&layout=embedded`;
```

Reply:
@murtaza98 are you sure doing this would resolve? Did you try this? According to my testing experience, I consciously removed the `/` in the URL. It needs to be removed [here](https://github.com/RocketChat/Rocket.Chat.Livechat/blob/c438eb2ee9a2c0d31f82a1bfaa8dd65888378b5d/src/components/Calls/JoinCallButton.js#L23) for it to work.

Reply:
Yups, This did solve my issue when I was testing directly by installing new widget on core i.e. `{serverUrl}/livechat` route.

Also sidenote: Usually we don't allow any extra slash at the back of any base URL (eg, in `Site_Url` setting on RC) and the livechat was built on this assumption. For more info, see the logic [here](https://github.com/RocketChat/Rocket.Chat.Livechat/blob/4f60b934e367ffb21152c37e2608bbe660e35ea5/src/api.js#L4)